modules/guides/examples/javascript-udfs/create-scoped-javascript-function-with-rest.sh

```diff
@@ -1,4 +1,4 @@
-curl -v -X POST  'http://localhost:8093/evaluator/v1/libraries/my-library?bucket=travel-sample&scope=inventory' \
+curl -v -X POST  'http://localhost:8093/evaluator/v1/libraries/my-library?bucket=$DATABASE_NAME&scope=samples' \
```

Comment:
I think you need double quotes to allow environment variable expansion, and braces to delimit the end of the environment variable

```suggestion
curl -v -X POST  "http://localhost:8093/evaluator/v1/libraries/my-library?bucket=${DATABASE_NAME}&scope=samples" \
```

Reply:
TBH this example file is not needed at the moment, javascript udfs are currently embargoed in Elixir

Reply:
updated